src/server/rest/v1/service/TagService.ts

```diff
@@ -562,4 +643,24 @@ export default class TagService {
     );
     return result;
   }
+
+  private static async importTag(action: ServerAction, req: Request, tag: any): Promise<void> {
+    try {
+      const newUploadedTag: ImportedTag = {
+        id: tag.id.toUpperCase(),
+        description: tag.description ? tag.description : `Tag ID '${tag.id}'`,
+        importedBy: req.user.id,
+        status: ImportStatus.UNKNOWN
+      };
+      await TagStorage.saveImportedTag(req.user.tenantID, newUploadedTag);
```

Comment:
We should check the data with ajv (performant) before we save it in the DB

Reply:
Done